packages/angular-cli/tasks/server/universal-dev-server.ts

```diff
@@ -0,0 +1,64 @@
+import * as path from 'path';
+const nodemon = require('nodemon');
+const tinyLr = require('tiny-lr');
+
+export class UniversalDevServer {
+  private appPath: string = '';
+  private nodemonConfig: any;
+  private demon: any;
+  private liveServer: any;
+
+  constructor(private webpackCompiler: any, private webpackDevServerConfiguration: any) {
```

Comment:
If the concern is trying to hide the `webpack-dev-middleware` from the user, I'd rather see that being used in server.ts and then `packaged/blackboxed` universal module which runs the dev-middleware